api/admin/routes.py

```diff
@@ -463,30 +557,126 @@ def resolve_complaints(identifier_type, identifier):
 def edit_classifications(identifier_type, identifier):
     return app.manager.admin_work_controller.edit_classifications(identifier_type, identifier)
 
+
 @app.route('/admin/roles')
 @allows_cors(allowed_domain_type=set({"admin"}))
 @returns_json_or_response_or_problem_detail
 def roles():
+    """Return a mapping from MARC codes to contributor roles.
+    ---
+    get:
+      tags: 
+        - administration
+      description: Returns a JSON of properly mapped MARC codes
```

Comment:
a nit, but add "that are currently defined in this system" or something like that?